src/style.css

```diff
@@ -1 +1,35 @@
 /* add styles here */
+:root{
+  --main-images: (./images/star-active.svg)
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
The `box-sizing: border-box;` should be applied globally to all elements for consistency. Move it to a universal selector `*` at the top of your stylesheet.